packages/core/src/internal/decorators/animate.ts

```diff
@@ -14,6 +14,7 @@ import { runPropertyAnimations } from '../motion/utils.js';
 
 // https://www.typescriptlang.org/docs/handbook/decorators.html#class-decorators
 export function animate(config: PropertyDrivenAnimation) {
+  // eslint-disable-next-line @typescript-eslint/ban-types
   return function _DecoratorName<T extends { new (...args: any[]): {} }>(constr: T) {
```

Comment:
There doesn't seem to be a way to type this properly and the updated eslint does not like the {} object as a type.